boards/arm/stm32f3_disco/stm32f3_disco_E.overlay

```diff
@@ -22,3 +22,13 @@
 		label = "LSM303AGR-ACCEL";
 	};
 };
+
+&spi1 {
+	cs-gpios = <&gpioe 3 GPIO_ACTIVE_LOW>;
+    i3g4250d@0 {
+        compatible = "st,i3g4250d";
+        reg = <0>;
+        spi-max-frequency = <1000000>;
+        label = "I3G4250D";
+    };
+};
```

Comment:
Please indent with tabs instead of spaces

Reply:
> Please add this sensor to `tests/drivers/build_all/sensor`

Ah nice this might replace my custom test definition in `tests/drivers/sensor/..`? Or is only a build test, to check for compile issues?

Reply:
`tests/drivers/build_all/sensor` is a build test to check for compile issues

Reply:
added to `build_all` spi device tree